artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -116,7 +118,7 @@ public void stop() {
   public final void handleException(
       final Throwable exception, final SubscriberExceptionContext context) {
     if (!isSpecFailure(exception)) {
-      logger.log(Level.FATAL, "Unexpected exception thrown in handler - PLEASE FIX OR REPORT");
+      LOG.error("Unhandled exception in handler for event " + context.getEvent(), exception);
```

Comment:
The `ALogger` in this case will wind up going to stdout whereas `LogManager.getLogger` won't.  I'd be tempted to remove the `ALogger` from here and always use Log4J directly but then edit the default log4j.xml to send logs from this class to stdout. Or if you prefer, add a method to `ALogger` so that it can print exception stack traces and stick with using it.

Reply:
Added some `ALogger` methods that accept exceptions.